cache_images/podman_tooling.sh

```diff
@@ -6,14 +6,28 @@
 # as root or through sudo.  The script should not be used for any other
 # purpose or from any other context.
 
-set -e
+set -eo pipefail
 
 SCRIPT_FILEPATH=$(realpath "${BASH_SOURCE[0]}")
 SCRIPT_DIRPATH=$(dirname "$SCRIPT_FILEPATH")
 REPO_DIRPATH=$(realpath "$SCRIPT_DIRPATH/../")
 # shellcheck source=./lib.sh
 source "$REPO_DIRPATH/lib.sh"
 
+if [[ "$OS_RELEASE_ID" == "ubuntu" ]]; then
+    if [[ -n "$(type -P bats)" ]]; then
+        die "Bats _MUST_ not be installed on ubuntu until fixed: https://bugs.launchpad.net/ubuntu/+source/bats/+bug/1882542"
+    fi
+    bats_version="1.7.0"
+    dl_url="https://github.com/bats-core/bats-core/archive/v${bats_version}.tar.gz"
+    echo "Installing bats $bats_version"
+    curl --fail --location "$dl_url" | tar xzv -C /tmp
```

Comment:
After reading the [ubuntu log](https://cirrus-ci.com/task/6658219291443200?logs=main#L1711) I belatedly realized that the `-v` is a bit excessive... but not worth a re-push over. Will lgtm-hold so you can test the images before merging. Thanks @cevich 

/lgtm
/hold

Reply:
Oh, this repo doesn't work that way :-). LGTM anyway.

Reply:
Sounds good, if this PR needs re-pushing for new images, I can fix that up quick.

Reply:
fixed.